test/parallel/test-runner-enable-source-maps-issue.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+require('../common');
+const assert = require('node:assert');
+const { spawnSync } = require('node:child_process');
+const { test } = require('node:test');
+const fixtures = require('../common/fixtures');
+const skipIfNoInspector = {
```

Comment:
I'm not sure that we actually need this.

Reply:
Let me remove this.